docs/make-a-widget.md

```diff
@@ -138,6 +138,13 @@ Follow these steps for each of the predefined widget types described in this doc
   </value>
 </portlet-preference>
 
+#### Configuring your list of links from a URL
+
+* Rather than hardcode your links into a portlet preference, you can point to a URL and get your links dynamically. 
+1. Omit the "links" entry in the widgetConfig JSON.
+2. In widgetConfig, add the following:
+   "getLinksURL": "true"
```

Comment:
What happens in the case where we have a valid widgetURL and also links in the widgetConfig?